tests/sentry/api/endpoints/test_organization_events_trends_v2.py

```diff
@@ -124,7 +124,7 @@ def test_simple_with_trends(self, mock_detect_breakpoints):
         assert events["data"] == mock_trends_result
 
         assert len(result_stats) > 0
-        assert len(result_stats.get(f"{self.project.slug},foo", [])) > 0
+        assert len(result_stats.get(f"{self.project.id},foo", [])) > 0
```

Comment:
The response here does change slightly but it's operating as if that dev only mode thing is always on and i verified that works as is as we also modify the project value in the trends portion of the payload to the id.